### ISSUE TITLE
Error conecting

### ISSUE THREAD

**Issue by askthomas** [bimmerconnected/bimmer_connected#354]
### Describe the issue

Intergration ger erro 400

Retrying setup: 400 Client Error: Bad Request for url: https://customer.bmwgroup.com/gcdm/oauth/authenticate

I dont think i change anything, it just starded, i double check accounts, and they wokrks on bmw www site.

### Expected behavior

Get car status in to HAS

### Which Home Assistant version are you using?

core-2021.11.4

### What is your region?

Rest of world

### ConnectedDrive website

- [X] I can still successfully login to the BMW ConnectedDrive website and the car status is available there.
- [X] I have ConnectedDrive enabled for my vehicle.

### Number of cars

- [ ] I have 2 or more cars linked to the ConnectedDrive account.
- [ ] I have a Mini vehicle linked to my account.

### Output of bimmer_connected fingerprint

```txt
DEBUG:bimmer_connected.account:Getting vehicle list
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-ucs/v1/presentation/oauth/config HTTP/1.1" 200 580
DEBUG:bimmer_connected.account:Authenticating against GCDM with MyBMW flow.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): customer.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/authenticate HTTP/1.1" 200 411
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/authenticate HTTP/1.1" 302 0
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/token HTTP/1.1" 200 1963
DEBUG:bimmer_connected.account:got new token wRyAgTsDsFrNXo6Tw131AYV7AYg with expiration date 2021-11-21 10:14:37.188419DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1637478878.195579&tireGuardMode=ENABLED HTTP/1.1" 200 8587
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1637478880.304321&tireGuardMode=ENABLED HTTP/1.1" 200 2
Getting 'charging-sessions' for WBA6N3101MFK67711
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-chs/v1/charging-sessions
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-chs/v1/charging-sessions?vin=WBA6N3101MFK67711&maxResults=40&include_date_picker=true HTTP/1.1" 200 None
Getting 'charging-statistics' for WBA6N3101MFK67711
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-chs/v1/charging-statistics
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-chs/v1/charging-statistics?vin=WBA6N3101MFK67711&currentDate=2021-11-21T08%3A14%3A42.104677 HTTP/1.1" 200 None
fingerprint of the vehicles written to /root/vehicle_fingerprint/2021-11-21_09-14-37
```

### Anything in the logs that might be useful for us?

```txt
It stared that all values not working, and now error to connect
```

**Comment by nottledim** [comment]
You probably need to confirm you've installed the new custom component myBMW because the ha integration no longer works.
```
https://github.com/bimmerconnected/ha_custom_component/releases/tag/20211113.1
```

**Comment by askthomas** [comment]
Yes, I had not installed the new Custom Componment in HACS, After that it works agan, Thanks for help.

**Comment by zhaozengxiao** [comment]
When can you support China?

**Comment by rikroe** [comment]
Closing it was fixed within the library/custom component and is included in the HA 2021.12 beta, so it will be released in HA on Dec. 11th.
